app/user/invite-users/invite-users.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component } from '@angular/core';
+import { ToasterService } from 'angular2-toaster';
+import { TranslateService } from 'ng2-translate';
+
+import { UserService } from '../../core/user/user.service';
+import { User } from '../../core/user/user.model';
+
+@Component({
+  moduleId: module.id,
+  selector: 'hip-invite-users',
+  templateUrl: 'invite-users.component.html',
+  styleUrls: ['invite-users.component.css']
+})
+export class InviteUsersComponent {
+  
+  emails: String[] = [];
+  users: User[] = [];
+  canSend = false;
+  errorItem: string = "";
+  isError = false;
+  translatedResponse: any;
+
+  constructor(private toasterService: ToasterService,
+              private userService: UserService,
+              private translateService: TranslateService) {
+  }
+
+  public validateEmail(item: any): string {
```

Comment:
still no valid email validation.
a mail like m.h@somthing.com or username@mail.uni-paderborn.de are valid email addresses, but get validated wrong.

You can find here valid regex for emails: http://emailregex.com/